iOS Example Frame SPM/iOS Example Frame SPM.xcodeproj/xcshareddata/xcschemes/iOS Example Frame.xcscheme

```diff
@@ -44,6 +44,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "IDELaunchSchemeLanguageRightToLeftLayoutDirection"
```

Comment:
Did you intend to commit this?

Reply:
```suggestion
```

Reply:
Very creative way to fix. I wasn't sure if it was intended or not so I wasn't confident to propose code change. But I need to remember this convenient _hack_ when dealing with non code files 😬